tests/unit/worker/test_submission_worker.py

```diff
@@ -1,28 +1,77 @@
 import boto3
+import mock
 import os
 import shutil
 import tempfile
 
+from datetime import timedelta
 from moto import mock_sqs
 from os.path import join
-from unittest import TestCase
 
+from django.contrib.auth.models import User
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.utils import timezone
+
+from rest_framework.test import APITestCase
+
+from challenges.models import (
+    Challenge,
+    ChallengePhase,
+)
+from hosts.models import ChallengeHostTeam
+from jobs.models import Submission
+from participants.models import ParticipantTeam
 from scripts.workers.submission_worker import (
     create_dir,
     create_dir_as_python_package,
+    extract_submission_data,
     return_file_url_per_environment,
     get_or_create_sqs_queue
 )
 
 
-class BaseAPITestClass(TestCase):
+class BaseAPITestClass(APITestCase):
     def setUp(self):
+
         self.BASE_TEMP_DIR = tempfile.mkdtemp()
+        self.COMPUTE_DIRECTORY_PATH = join(
+            self.BASE_TEMP_DIR, "compute"
+        )
+        self.CHALLENGE_DATA_BASE_DIR = join(
+            self.COMPUTE_DIRECTORY_PATH, "challenge_data"
+        )
+        self.SUBMISSION_DATA_BASE_DIR = join(
+            self.COMPUTE_DIRECTORY_PATH, "submission_files"
+        )
+        self.CHALLENGE_DATA_DIR = join(
+            self.CHALLENGE_DATA_BASE_DIR, "challenge_{challenge_id}"
+        )
+        self.PHASE_DATA_BASE_DIR = join(
+            self.CHALLENGE_DATA_DIR, "phase_data"
+        )
+        self.PHASE_DATA_DIR = join(
```

Comment:
@nikochiko rather than coding in this way you can add helper methods which will take `phase_id` as input and return the complete path. I don't think we will need `PHASE_DATA_BASE_DIR` var separately. You can write methods to get path based on `phase_id` and `submission_id`